google-cloud-testing/pubsub-benchwrapper/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>benchwrapper</artifactId>
```

Comment:
```suggestion
  <artifactId>pubsub-benchwrapper</artifactId>
```

Reply:
Does it matter? We just `mvn exec:run` this.

Reply:
Long term it won't matter when the repo is fully split, but for now, we have to install the code locally with `mvn install`. This `artifactId` collides with the storage benchwrapper so it will either fail to install or one will overwrite the other and cause failures.

Reply:
TIL! Thank you.

Reply:
It mattered and broke the build. Still debugging exactly how but this should be rolled back or fixed ASAP. :-(